test/SlowTests/Issues/RavenDB_2793.cs

```diff
@@ -1,76 +0,0 @@
-﻿using System.Collections.Generic;
-using System.Linq;
-using FastTests;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.Issues
-{
-    public class RavenDB_2793 : RavenTestBase
-    {
-        private class Person
-        {
-            public string FirstName { get; set; }
-
-            public string LastName { get; set; }
-        }
-
-        private class People_FirstName : AbstractTransformerCreationTask<Person>
-        {
-            public class Result
-            {
-                public string FirstName { get; set; }
-            }
-
-            public People_FirstName()
-            {
-                TransformResults = people => from person in people
-                                             select new
-                                             {
-                                                 FirstName = person.FirstName + Parameter("value").Value<int>()
```

Comment:
Restore candidate: make sure we properly handle numeric parameters in transformation function